h2integrate/storage/battery/pysam_battery.py

```diff
@@ -113,6 +117,21 @@ def setup(self):
         self.config = PySAMBatteryPerformanceModelConfig.from_dict(
             merge_shared_inputs(self.options["tech_config"]["model_inputs"], "performance")
         )
+
+        self.add_input(
+            "charge_rate",
+            val=self.config.system_capacity_kw,
+            units="kW",
+            desc="Battery charge rate",
+        )
+
+        self.add_input(
+            "storage_capacity",
```

Comment:
I think `battery_energy_kwh` is more clear, unless we need it to be generic for swapping out other storage systems with the controls etc.